scroller/package.json

```diff
@@ -6,8 +6,5 @@
     "@angular/common": ">=12.0.0",
     "@angular/core": ">=12.0.0",
     "vscroll": "^1.5.4"
-  },
-  "dependencies": {
-    "tslib": "^2.3.1"
```

Comment:
Removed because it's added automatically by `ng-packagr` when the library is being built.